.pre-commit-config.yaml

```diff
@@ -32,3 +32,11 @@ repos:
     rev: 'v0.2.0'
     hooks:
       - id: spotlessCheck
+
+  - repo: local
+    hooks:
+      - id: run-npm-lint
+        name: Lint API tests
+        entry: bash -c 'cd api-tests && npm run lint'
+        language: system
+        pass_filenames: false
```

Comment:
Is it possible to filter this so it only runs if you actually change something in `api-tests`?

Reply:
Turns out yes!